### PR TITLE
docs: add package manager tabs to CI

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Adds `npm` `yarn` and `pnpm` tabs for all of our CI guides

### CODE REVIEW

File: docs/pages/docs/ci/github-actions.mdx [hunk 1]
```diff
@@ -3,56 +3,191 @@ title: Using Turborepo with GitHub Actions
...
-      timeout-minutes: 15
-      runs-on: ${{ matrix.os }}
-      # To use Remote Caching, uncomment the next lines and follow the steps below.
-      # env:
-      #  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-      #  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
-      strategy:
-        matrix:
-          os: [ubuntu-latest, macos-latest]
+For a given root `package.json`:
```
[the remark this answers is not in the document]
Comment (Ethan-Arrowood) [reply docs/pages/docs/ci/github-actions.mdx:12]: These would already exist, but are provided as context for the CI file which makes calls to the scripts `build` and `test`

File: docs/pages/docs/ci/github-actions.mdx [hunk 2]
```diff
@@ -3,56 +3,191 @@ title: Using Turborepo with GitHub Actions
...
-        - uses: pnpm/action-setup@v2.0.1
-          with:
-            version: 6.32.2
+```json
+{
+  "scripts": {
+    "build": "turbo run build",
+    "test": "turbo run test"
+  },
+  "dependencies": {
```
Comment (jaredpalmer) [inline docs/pages/docs/ci/github-actions.mdx:20]: `devDependencies` for consistency

Comment (jaredpalmer) [reply docs/pages/docs/ci/github-actions.mdx:20]: add a "name" field to each `package.json`

Comment (jaredpalmer) [reply docs/pages/docs/ci/github-actions.mdx:20]: `"name": "my-turborepo"`

**Review by jaredpalmer — APPROVED** [review]

**Pull request merged**